### ISSUE TITLE
Store files with an .eml extension

### ISSUE THREAD

**Issue by ad-si** [OfflineIMAP/offlineimap#681]
Why aren't the emails stored with an `.eml` file extension? This would allow me to easily open them and even see their content already in the preview (macOS).

**Comment by echodreamz** [comment]
Agreed... Would be great to have a way to just dump emails to .eml files.
